source/guide_sulu.rst

```diff
@@ -0,0 +1,209 @@
+
+``cd`` to one level above your :manual:`DocumentRoot <web-documentroot>`, then use the PHP Dependency Manager Composer_ to create a new project based on the **Sulu Skeleton**.
+
+.. note:: Replace ``sulucms`` in the examples with your desired name for the directory/database.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ composer create-project sulu/skeleton sulucms
+ Creating a "sulu/skeleton" project at "./sulucms"
+ Installing sulu/skeleton (2.0.7)
+   - Installing sulu/skeleton (2.0.7): Loading from cache
+ Created project in /var/www/virtual/ckrack/sulucms
+ […]
+ [isabell@stardust isabell]$
+
+Remove your empty :manual:`DocumentRoot <web-documentroot>` and create a new symbolic link to the ``sulucms/public`` directory.
+
+.. warning:: Make sure ``html`` is empty before deleting it. If there are any files you want to keep
+   in ``html``, you can also rename the folder instead of deleting it.
+
+.. code-block:: console
+
+ [isabell@stardust isabell]$ rmdir html
+ [isabell@stardust isabell]$ ln -s /var/www/virtual/$USER/sulucms/public html
+ [isabell@stardust isabell]$ cd ~
+ [isabell@stardust ~]$
+
+
+Configuration
+=============
+
+During the setup process you need to configure an ``.env.local`` file with database credentials. We suggest you use an :manual_anchor:`additional database <database-mysql.html#additional-databases>` for Sulu CMS to save your data. You have to create this database first using the following command.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mysql -e "CREATE DATABASE ${USER}_sulucms"
+ [isabell@stardust ~]$
+
+.. note:: As uberspace uses MariaDb, you need to find the version and prefix the serverVersion with mariadb.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mysql -e "SELECT VERSION()"
+  +-----------------+
+  | VERSION()       |
+  +-----------------+
+  | 10.3.22-MariaDB |
+  +-----------------+
+ [isabell@stardust ~]$
+
+Review the sample configuration file ``.../sulucms/.env``. Then edit the ``.env.local`` file and change the values of ``APP_ENV`` and ``DATABASE_URL``, to reflect your MySQL :manual_anchor:`credentials <database-mysql.html#login-credentials>` and save the file.
+
+.. code-block:: ini
+
+ APP_ENV=prod
+ DATABASE_URL=mysql://isabell:MySuperSecretPassword@localhost/isabell_sulucms?serverVersion=mariadb-10.3.22
+
+.. note:: You can optionally configure Sulu to send emails in the same place.
+
+
+When you’re done with the configuration, populate the database with Sulu’s default data by following the build walkthrough.
+
+.. code-block:: console
+ :emphasize-lines: 31
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/sulucms
+ [isabell@stardust sulucms]$ ./bin/adminconsole sulu:build dev
```

Comment:
Since `APP_ENV` is `prod`, should we also use `prod` instead of `dev` here?

Reply:
Doing that seems to skip user creation, so the additional commands `./bin/adminconsole sulu:security:role:create` and `./bin/adminconsole sulu:security:user:create` are needed.

Reply:
This also forces users to choose proper password instead of logging in using `admin:admin`, so it seems like the better way to do it. What do you think?

Reply:
Yeah, I agree. I know too many people that don't change their default passwords, too..

Reply:
This still says "dev". Is that on purpose?

Reply:
Nope! Fixed it